public/gameInfo.php

```diff
@@ -586,9 +586,9 @@ function () {
 
                 if( isset( $user ) )
                 {
-                    echo "<div class='devbox'>";
-                    echo "<span onclick=\"$('#devboxcontent').toggle(500); return false;\">More info (Click to show):</span><br/>";
-                    echo "<div id='devboxcontent'>";
+                    echo "<div class='userinfobox'>";
+                    echo "<span onclick=\"$('#userinfoboxcontent').toggle(500); return false;\">More info (Click to show):</span><br/>";
+                    echo "<div id='userinfoboxcontent'>";
```

Comment:
Needed to change classes here. Using `devbox` creates a problem with the `Dev (Click to show)` menu. In order to use both toggle-able menus at once, they need to be separate classes.